### PR TITLE
Fixed broken link in README.md

### LINKED ISSUE #147: [README] Broken link to image
**Description**
This repo's readme.md has a broken link to the first image.

**Expected Behavior**
The image that should be displayed can be found here - https://github.com/layer5io/meshery-operator/tree/master/img/readme

### PR DESCRIPTION
**Pull request by DineshKoushik**
**Description**

This PR fixes #147 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by DineshKoushik** [comment]
@leecalcote oops what am I missing?

**Comment by leecalcote** [comment]
@DineshKoushik you're not missing anything here. Looks good.

**Comment by leecalcote** [comment]
See the link to join Slack above

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/layer5/layer5-tag-white-bg.png" width="45%" /></p>
+  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/layer5-tag-white-bg.png" width="45%" /></p>
```
Comment (leecalcote) [inline README.md:2]:
```suggestion
  <img align="center" src="https://raw.githubusercontent.com/layer5io/meshery-operator/master/img/readme/meshery-logo-light-text.svg" width="45%" /></p>
```

**Review by leecalcote — APPROVED** [review]
Thanks, @DineshKoushik

**Pull request merged**